src/devices/QwiicButton/QwiicButton.InterruptConfiguration.cs

```diff
@@ -0,0 +1,92 @@
+﻿//// Licensed to the .NET Foundation under one or more agreements.
+//// The .NET Foundation licenses this file to you under the MIT license.
+//// See the LICENSE file in the project root for more information.
+
+using Iot.Device.QwiicButton.RegisterMapping;
+
+namespace Iot.Device.QwiicButton
+{
+    public sealed partial class QwiicButton
+    {
+        /// <summary>
+        /// When called, the interrupt will be configured to trigger when the button is pressed.
+        /// If <see cref="EnableClickedInterrupt"/> has also been called,
+        /// then the interrupt will trigger on either a push or a click.
+        /// </summary>
+        public void EnablePressedInterrupt()
```

Comment:
This and below should be a property (i.e. `PressedInterruptEnabled`)

Reply:
That would clash with MS' own design guidelines stating that: "Properties are meant to be used like fields, meaning that properties should not be computationally complex or produce side effects".
Read more here: https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/ms229054(v=vs.100)?redirectedfrom=MSDN